probeinfo_api.yaml

```
@@ -734,6 +750,125 @@ components:
         If not specified, retention will be unlimited.
       type: integer
 
+    OverrideAttributes:
+      type: array
+      description: |
+        Currently not in use.
+        Mappings of Pub/Sub attribute names to static values; these are applied in the Decoder
+        immediately before incorporating metadata into the payload, so can be used to overwrite values calculated in
+        the pipeline; a null value will cause the pipeline to drop the named attribute; some attribute names differ
+        from the nested metadata format in BigQuery, so for example you must use \"geo_city\" here in order to
+        manipulate the value that shows up as metadata.geo.city; implemented for bug 1742172.
+      items:
+        type: object
+        additionalProperties: false
+        properties:
+          name:
+            type: string
+            required: true
+            enum:
+              - geo_city
+              - geo_subdivision1
+              - geo_subdivision2
+              - normalized_channel
+          value:
+            type: [ string, null ]
+            required: true
+
+    ExpirationPolicy:
+      type: object
+      description: |
+        Currently not in use.  Various options controlling data lifecycle.
+      additionalProperties: false
+      properties:
+        collect_through_date:
+          type: string
+          description: |
+            Currently not in use.  If present, the pipeline will reject new data with submission_timestamp after the
+            given date, sending it to error output.
+          pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
```

```suggestion
          pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
          example: "2025-12-31"
```

For string types, redoc generates `"string"` as the example field, but you can provide a more specific example this way and it will get rendered in the docs.